data/api/v1/full_spec.yaml

```diff
@@ -7943,7 +7943,7 @@ components:
       - DOUBLE
     TimeseriesWidgetDefinition:
       description: The timeseries visualization allows you to display the evolution
-        of one or more metrics, log events, or Analyzed Spans over time.
+        of one or more metrics, log events, or Indexed Spans over time.
```

Comment:
These changes must be made in the API spec repo instead (can to talk with Sarina or Kaylyn about how)